### PR TITLE
fix some function of getNextSiblings/ isFunction/FadeIn

### PR DESCRIPTION
**Pull request by catchonme**
Errors:
1. `getAllSiblings` return all siblings of target element, not return next siblings of target element
2. `Object.prototype.toString(item)` always return `[object Object]`
3. Use `const` to define a variable shouldn't change its value

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,15 +162,21 @@ In place of common selectors like class, id or attribute we can use `document.qu
...
-    function getAllSiblings(elem, filter) {
-      var sibs = [];
-      elem = elem.parentNode.firstChild;
-      do {
-          if (elem.nodeType === 3) continue; // ignore text nodes
-          if (!filter || filter(elem)) sibs.push(elem);
-      } while (elem = elem.nextSibling)
-    return sibs;
-    }
+    function getNextSiblings(elem, filter) {
```
Comment (camsong) [inline README.md]: Actually this return all siblings, not just 'next' siblings, can you update this?
and we prefer two space for indentation.

File: README.md [hunk 2]
```diff
@@ -927,7 +933,7 @@ Most of jQuery utilities are also found in the native API. Other advanced functi
     if (typeof item === 'function') {
       return true;
     }
-    var type = Object.prototype.toString(item);
+    var type = Object.prototype.toString.call(item);
```
Comment (camsong) [inline README.md]: good catch.

File: README.md (hunk 1 above)
Comment (catchonme) [reply README.md]: use `if (nextElem === elem.nextElementSibling)` can filter prevSiblings, so we can get nextSiblings, and I compare it with jQuery, and the result is the same.

**Comment by camsong** [comment]
Brilliant!

**Pull request merged**